openquake/commonlib/nrml_examples/source_model/mixed.xml

```diff
@@ -123,6 +123,22 @@ xmlns:gml="http://www.opengis.net/gml"
             <rake>
                 30.0
             </rake>
+            <hypoList>
+                <hypo alongStrike="0.25" downDip="0.25" weight="0.3"/>
+                <hypo alongStrike="0.75" downDip="0.75" weight="0.7"/>
+            </hypoList>
+            <slipList>
+                <slip
+                weight="0.7"
+                >
+                    90.0
+                </slip>
+                <slip
+                weight="0.3"
+                >
+                    135.0
+                </slip>
+            </slipList>
```

Comment:
I borrowed the format for the `hypo` nodes from the `hypocenter` nodes, but perhaps it was not a good idea. What would be a more consistent format? Perhaps

```
<hypo weight="0.3"/>
0.25 0.25
</hypo>
```

or
```
<hypo weight="0.3"/>
<alongStrike>0.25</alongStrike>
<downDip>0.25</downDip>
</hypo>
```
?